app/controllers/transportMeans/departure/IdentificationController.scala

```diff
@@ -56,27 +59,41 @@ class IdentificationController @Inject() (
     .requireData(lrn)
     .async {
       implicit request =>
+        val viewModel              = viewModelProvider(request.userAnswers, departureIndex)
+        def inlandMode: InlandMode = request.userAnswers.get(InlandModePage).getOrElse(InlandMode("", ""))
+        val para: String = (inlandMode.code, departureIndex) match {
+          case ("3", Index(0)) => viewModel.paragraph1
+          case ("3", _)        => viewModel.paragraph2
+          case _               => ""
+        }
```

Comment:
Can this logic live inside the view model? And can we import `config.Constants.ModeOfTransport.Road` and use that instead of `3`?

Reply:
Done